polls/templates/polls/auth_token.html

```diff
@@ -30,12 +32,54 @@ <h3>API Token</h3>
 headers = {
     "Authorization": "Token {{ token.key }}"
     }
-url = "{{ admin_api_url }}"
+url = "{{ admin_api }}"
 response = requests.get(url, headers=headers)
 print(response.json())
 print(response.status_code)
         </code>
       </pre>
+
+      <div class="mt-3">
+        <h4>Annotation API</h4>
+        <p>Fetches all entries of annotation table</p>
+        <pre>
+          <code>
+import requests
+
+headers = {
+    "Authorization": "Token {{ token.key }}"
+    }
+annotation_api = "{{ annotation_api }}"
+response = requests.get(annotation_api, headers=headers)
+print(response.json())
+print(response.status_code)
+          </code>
+        </pre>
+      </div>
+
+      <div class="mt-3">
+        <h4>Lock Users API</h4>
+        <p>Lock users based on posted list containing user ids</p>
+        <pre>
+          <code>
+import requests
+
+headers = {
+    "Authorization": "Token {{ token.key }}"
+    }
+# list of user ids
+user_ids = [10, 5]
```

Comment:
Hard codded user ids wouldn't always work except for the hardcoded users, since the aim is not to limit the auth lock to two users, then the id should be dynamic.